recipes/arize-phoenix-client/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "arize-phoenix-client" %}
+{% set version = "1.0.1" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/arize_phoenix_client-{{ version }}.tar.gz
+  sha256: d5e24b84a93b601aa101caaeab6678cc2a1a6a7317aaa5a820678f0852d9ae9c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - httpx
+    - typing_extensions
+
+test:
+  imports:
+    - phoenix.client
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: LLM Observability
+  home: https://github.com/Arize-ai/phoenix/tree/main/packages/phoenix-phoenix
```

Comment:
This url leads to a 404. Maybe use the one listed on pypi:
https://github.com/Arize-ai/phoenix

or:
https://github.com/Arize-ai/phoenix/tree/main/packages/phoenix-client